charts/puris/values.yaml

```diff
@@ -189,8 +189,10 @@ frontend:
         endpointReportedMaterialStocks: stockView/reported-material-stocks?ownMaterialNumber=
         # -- The endpoint for the partners' (customer) product stocks that they received from me
         endpointReportedProductStocks: stockView/reported-product-stocks?ownMaterialNumber=
-        # -- The endpoint for updating the partners product stocks and their material numbers for the stock view
+        # -- The endpoint for triggering an update of your material stocks on your partners side
```

Comment:
Please run helm docs and include the updated README

Reply:
Done